test/Microsoft.AspNet.Razor.Runtime.Test/project.json

```diff
@@ -1,6 +1,7 @@
 {
     "version": "1.0.0",
     "dependencies": {
+        "Moq": "4.2.1312.1622",
```

Comment:
surprised this project builds with this listed as a general dependency.  move to aspnet50 section.